### ISSUE TITLE
Switch automation to use pre-installed virtual environment

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#312]
In the off case that the CUDA linked to the warp-ctc virtual environment is causing our crash, independent of weather the warp ctc algorithm is actually used in this virtual environment, we should switch automation to use the pre-installed virtual environment.

**Comment by kdavis-mozilla** [comment]
This involves two steps, changing the documentation and the cron tab (Also maybe making sure the pre-installed virtual environment has all the right pip installs.)

**Comment by kdavis-mozilla** [comment]
Commit 3e42504 changed the documentation

**Comment by kdavis-mozilla** [comment]
The crontab has also been changed
```
SHELL=/bin/bash
PATH=/usr/local/bin:/usr/bin/:/bin
# Run WER every 15 mins
#*/15 *  *   *   *    test ! -f $HOME/.cache/deepspeech_wer/lock && (rm $HOME/.deepspeech_wer.locked.log; mkdir -p $HOME/wer && cd $HOME/wer && source /home/alissy/tf-warpctc-venv/bin/activate && /usr/bin/curl -H "Cache-Control: no-cache" -L https://raw.githubusercontent.com/mozilla/DeepSpeech/master/util/automation.py | ds_website_username="deepspeech1234" ds_website_privkey="$HOME/.ssh/website_key" ds_website_server_fqdn="doyouunderstand.me" ds_website_server_root="www/" ds_gpu_usage_root="/data/automation/gpu/" ds_wer_automation="./bin/run-wer-automation.sh" python -u ; cd) 2>$HOME/.deepspeech_wer.err.log | /usr/bin/ts "\%Y-\%m-\%d \%H:\%M:\%S" > $HOME/.deepspeech_wer.out.log || TZ='Europe/Berlin' date --rfc-2822 >> $HOME/.deepspeech_wer.locked.log
*/15 *  *   *   *    test ! -f $HOME/.cache/deepspeech_wer/lock && (rm $HOME/.deepspeech_wer.locked.log; mkdir -p $HOME/wer && cd $HOME/wer && source /usr/local/tensorflow-env/bin/activate && /usr/bin/curl -H "Cache-Control: no-cache" -L https://raw.githubusercontent.com/mozilla/DeepSpeech/master/util/automation.py | ds_website_username="deepspeech1234" ds_website_privkey="$HOME/.ssh/website_key" ds_website_server_fqdn="doyouunderstand.me" ds_website_server_root="www/" ds_gpu_usage_root="/data/automation/gpu/" ds_wer_automation="./bin/run-wer-automation.sh" python -u ; cd) 2>$HOME/.deepspeech_wer.err.log | /usr/bin/ts "\%Y-\%m-\%d \%H:\%M:\%S" > $HOME/.deepspeech_wer.out.log || TZ='Europe/Berlin' date --rfc-2822 >> $HOME/.deepspeech_wer.locked.log
```

**Comment by kdavis-mozilla** [comment]
Installed pip packages not in the pre-installed virtual environment but in the warpctc virtual environment 
```
pip install cycler
pip install gitdb2
pip install GitPython
pip install matplotlib
pip install natsort
pip install smmap2
```

**Comment by kdavis-mozilla** [comment]
Fixed with b751428
